test/cppsuite/src/util/options_parser.cpp

```diff
@@ -0,0 +1,64 @@
+/*-
+ * Public Domain 2014-present MongoDB, Inc.
+ * Public Domain 2008-2014 WiredTiger, Inc.
+ *
+ * This is free and unencumbered software released into the public domain.
+ *
+ * Anyone is free to copy, modify, publish, use, compile, sell, or
+ * distribute this software, either in source code form or as a compiled
+ * binary, for any purpose, commercial or non-commercial, and by any
+ * means.
+ *
+ * In jurisdictions that recognize copyright laws, the author or authors
+ * of this software dedicate any and all copyright interest in the
+ * software to the public domain. We make this dedication for the benefit
+ * of the public at large and to the detriment of our heirs and
+ * successors. We intend this dedication to be an overt act of
+ * relinquishment in perpetuity of all present and future rights to this
+ * software under copyright law.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ * OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "options_parser.h"
+
+/*
+ * option_exists --
+ *     Return whether an option has been provided on the command line.
+ */
+bool
+option_exists(const std::string &opt, int argc, char *argv[])
+{
+    for (int i = 0; i < argc; i++) {
+        auto arg = std::string(argv[i]);
+        auto result = arg.find(opt);
+        if (result != std::string::npos)
+            return true;
+    }
+    return false;
+}
+
+/*
+ * value_for_opt --
+ *     Return the value for a given option.
+ */
+std::string
+value_for_opt(const std::string &opt, int argc, char *argv[])
```

Comment:
Is this something that should/could be used by `run.cpp`? If so I reckon make another ticket to add it there.

Reply:
It is! I'll spin up a ticket.

Reply:
3d3b80a3e